dashboard/src/features/accessControl/selectors.js

```diff
@@ -2,17 +2,20 @@ import { createSelector } from 'reselect'
 import { policyOptions } from './constants'
 
 export const getPolicyNames = createSelector(
-  item => item.policies,
-  policies => policies.map(
-    policy => policyOptions.find(
-      elem => elem.value == policy
-    ).label
-  )
-)
+  item => item.grants,
+  grants => grants.map(
+    grant => {
+      let isProtected = grant.protected
```

Comment:
Super nit: if we're adding `const`, might as well make these two vars `const` as well?